client/rest/resources/rest.rosetta.json

```diff
@@ -0,0 +1,48 @@
+{
+  "network": {
+    "name": "testnet",
+    "description": "catapult public test network"
+  },
+
+  "port": 4000,
+  "protocol": "HTTPS",
+  "sslKeyPath": "",
+  "sslCertificatePath": "",
+  "crossDomain": {
+    "allowedHosts": ["*"],
+    "allowedMethods": ["GET", "POST", "PUT", "OPTIONS"]
+  },
+
+  "restEndpoint": "http://localhost:3000",
```

Comment:
I am not sure if it's possible to call the HTTP endpoint from HTTPS.

Reply:
in that case, we'd need to update it as part of deployment